### PR TITLE
Fix issues with reference-test to enable correct async behavior in StubLoader

### PR DESCRIPTION
**Pull request by lindner**

### CODE REVIEW

**Comment by lindner** [comment]
fyi to @shans

File: src/runtime/tests/reference-test.ts [hunk 1]
```diff
@@ -54,7 +54,7 @@ describe('references', () => {
 
   it('exposes a dereference API to particles', async () => {
     const loader = new StubLoader({
-      manifest: `
+      'manifest': `
```
Comment (mmandlis) [inline src/runtime/tests/reference-test.ts]: there is another unquoted manifest in a StubLoader constructor in arc-test.ts and other unquoted strings in manifest-test.ts - why does it break here?

Comment (lindner) [reply src/runtime/tests/reference-test.ts]: Oh, sorry, this was just an unrelated cleanup.

Comment (lindner) [reply src/runtime/tests/reference-test.ts]: The root cause of the breakage was that stubloader async caused something in Arc to be unresolved.  Adding the arc.idle call fixed it.

**Pull request merged**